lib/msal-browser/test/utils/TestTimeUtils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export function calculateExpiresDate(expiresIn: number): Date {
+    const totalExpiresInSeconds = Math.round(Date.now() / 1000 + expiresIn);
+    return new Date(totalExpiresInSeconds * 1000);
+}
+
+/**
+ * Returns current time in JS Date object (milliseconds) with offset in seconds
+ * @param offset
+ */
+export function nowDateWithOffset(offsetSeconds: number): Date {
```

Comment:
Optional: You can put these in the [shared test utils](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/shared-test-utils/src) so they don't need to be defined individually for each lib

Reply:
Thanks, I added it to shared test utils.